### PR TITLE
Add loader

### PR DESCRIPTION
**Pull request by jewsroch**
loader

### CODE REVIEW

**Comment by frojasg** [comment]
looks good to me :shipit: :+1:

File: PerDiem/PerDiem/Controllers/TransactionsViewController.m [hunk 1]
```diff
@@ -204,10 +224,14 @@ - (NSArray *)rightButtons
 
 - (void)swipeableTableViewCell:(SWTableViewCell *)cell didTriggerRightUtilityButtonWithIndex:(NSInteger)index {
     NSIndexPath *indexPath = [self.tableView indexPathForCell:cell];
+    NSInteger indexPathNumber = indexPath.row;
+    if (self.budget || self.period) {
```
Comment (frojasg) [inline PerDiem/PerDiem/Controllers/TransactionsViewController.m:228]: why did you add `self.period` ?

Comment (jewsroch) [inline PerDiem/PerDiem/Controllers/TransactionsViewController.m:228]: We don't want to show the loader in the transactions list when we're in the calendar/day view transition. `self.period` indicates that we're in a specific timeframe (day), right?

Comment (frojasg) [inline PerDiem/PerDiem/Controllers/TransactionsViewController.m:228]: this condition is to fix the indexPathNumber. I don't think it's related to the loader. is it?
